lib/taste_tester/ssh_util.rb

```diff
@@ -20,8 +20,11 @@ def ssh_base_cmd
           "-J #{TasteTester::Config.jumps}" : ''
         "#{TasteTester::Config.ssh_command} #{jumps} -T -o BatchMode=yes " +
           '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ' +
-          "-o ConnectTimeout=#{TasteTester::Config.ssh_connect_timeout} " +
-          "#{TasteTester::Config.user}@#{@host} "
+          "-o ConnectTimeout=#{TasteTester::Config.ssh_connect_timeout}"
+      end
+
+      def ssh_target
+        "#{TasteTester::Config.user}@#{@host}"
```

Comment:
we need to preserve the extra space after the host or this will fail because all the commands get appended in to it
```suggestion
        "#{TasteTester::Config.user}@#{@host} "
```

Reply:
doing that here seems a bit ... wrong. I'd rather do that on line 72.

Reply:
I'm fine with either option